core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright 2017 Adobe Systems Incorporated
+ *  Copyright 2015 Adobe Systems Incorporated
```

Comment:
@DeepaliAgrwl This merge looks worng.

Reply:
@rishabh-behl the file at thi slocation core/src/main/java/com/hcl/ecomm/core/schedulers/package-info.java has been removed. Please check and let us know if the issue persists